src/components/pickers/Menu/Menu.stories.tsx

```diff
@@ -253,6 +253,18 @@ export const MenuSelectableMultiple = (props) => {
   });
 };
 
+MenuSelectableMultiple.play = async ({ canvasElement }) => {
+  const { getByTestId } = within(canvasElement);
+
+  await userEvent.tab(getByTestId('Menu'));
+
+  await expect(getByTestId('MenuItem-1')).toHaveFocus();
```

Comment:
Running expect isn't necessary since we'll see these changes on the screenshot

Reply:
You mean don't run this check at all?

Reply:
Yep, but before make sure that you've captured the focused state in chromatic :)
If you wanna add a test to make sure the component is focused, add a unit test )

So the logic is:
1) If you're testing a visual state - use the storybook + play function
2) Otherwise, use unit tests

Reply:
@nikolaykost you removed only one expect of 2 :)

Reply:
I don't understand. Why we need to write play function but don't use expect in that functions?

Reply:
@nikolaykost AFAIK it's because chromatic will snapshot states of play function. If it changes it will let us know.